### ISSUE TITLE
Appveyor builds fine, but reports error to GitHub

### ISSUE THREAD

**Issue by bep** [gohugoio/hugo#3646]
This is annoying, as most PR builds are marked as red.

**Comment by anthonyfok** [comment]
I checked in Settings, and saw three active but duplicate webhooks to Appveyor (with different API keys),, and at least one of them was failing consistently:

1. Good one: https://ci.appveyor.com/project/bep/hugo/history
2. Bad one: https://ci.appveyor.com/project/bep/hugo-3wdka/history
3. ??

The Bad one usually finished first and told GitHub of the failed build, and GitHub was stuck with those red crosses.

Anyhow, out of the three, it seems the first one is the Good one, and I went ahead and unchecked the **Active** checkbox for the other two rogue ones.  Hope this fixes the issue, but let's observe for a few more days to make sure it is indeed back to normal before closing this issue.
